docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+-------------------
+Submitting a report
+-------------------
+
+.. _`abusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on
+or user on addons.mozilla.org.  Authentication is not required, but is recommended
+so reports can be responded to if nessecary.
+
+.. http:post:: /api/v3/abuse/report/
+
+    .. _abusereport-create-request:
+
+    :<json string|null addon: if reporting an add-on then the add-on id, slug, or guid of the add-on.
```

Comment:
I would rephrase this to "add-on id, slug, or guid if the abuse report relates to an add-on".

But which one is it? It's confusing that it could be any.